gctrpc/README.md

```diff
@@ -51,15 +64,12 @@ This will place three binaries in your `$GOBIN`;
 
 Make sure that your `$GOBIN` is in your `$PATH`.
```

Comment:
The last request I have also is to keep the installation section above the operating system directives as buf/gRPC enterprise gateway still needs the following packages installed:

```
go install \
    github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
    github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
    google.golang.org/protobuf/cmd/protoc-gen-go \
    google.golang.org/grpc/cmd/protoc-gen-go-grpc
 ```